Tests/Tests/APIClientSpec.swift

```diff
@@ -33,7 +32,14 @@ class APIClientSpec: QuickSpec {
 
                 client.perform(request) { result = $0 }
 
-                expect(result?.value).toEventually(equal("hello world"))
+                func value(_ result: Result<String, SwishError>?) -> String? {
+                  if case .success(let value)? = result {
+                    return value
+                  }
+                  return nil
+                }
+                
+                expect(value(result)).toEventually(equal("hello world"))
```

Comment:
I believe this might work as `try? result?.get()`, or even more preferably `try result?.get()` if `expect()` will accept a throwing autoclosure. If the autoclosure form doesn’t work, then an explicit closure definitely should, e.g.: `expect { try result?.get() }`.